app/src/main/java/com/example/ithaca_transit_android_v2/models/Stop.kt

```diff
@@ -0,0 +1,11 @@
+package com.example.ithaca_transit_android_v2.models
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+//Location. - Represents either a bus stop or a google place.
+@JsonClass(generateAdapter = true)
+// TODO: deprecate this class 
+data class Stop(
+    val stopID: String,
+    val lat: Double,
+    val long: Double
```

Comment:
use `Coordinate`

Reply:
Backend has `stop` in our current format. Their `lat` and `long` are in the same level with `stopID`. Maybe we should leave them as lat and long?

Reply:
right, it has the same issue as `Location`. it's fine to keep it as is